app/assets/javascripts/discourse/ember-cli-build.js

```diff
@@ -81,6 +81,11 @@ module.exports = function (defaults) {
     },
   });
 
+  // TODO: remove me
+  // Ember 3.28 still has some internal dependency on jQuery being a global,
+  // for the time being we will bring it in vendor.js
+  app.import("node_modules/jquery/dist/jquery.js", { prepend: true });
```

Comment:
Once we stop doing this, one difference will be that, when evaluated as a "real-ish" module, jQuery detects that and does not define the `window.$` and `window.jQuery` globals. We can add that back in the `jquery-plugins` initializer.

Currently on 3.28 that is not very feasible as the ember internals gets loaded before our initializer runs and expects the global to be defined. Once we are on 4+, that problem goes away. When we are in control of adding the `window.$` global may want to consider adding our own deprecation for it (asking to `import $ from "jquery";`) instead, which would pave the way for a better chance of us being able to eventually stop shipping jQuery in core. 

(At least we have some hope of statically scanning for the import, though the long term proper solution is to have plugins declare the dependency.)